gradle.properties

```diff
@@ -9,13 +9,13 @@
 #
 ### dependencies
 pravegaVersion=0.4.0-50.63aea88-SNAPSHOT
-flinkConnectorVersion=0.4.0-103.7971206-SNAPSHOT
+flinkConnectorVersion=0.4.0-100.d588197-SNAPSHOT
 
 ### Pravega-samples output library
 samplesVersion=0.4.0-SNAPSHOT
 
 ### Flink-connector examples
-flinkVersion=1.4.0
+flinkVersion=1.6.0
```

Comment:
Could you please confirm that existing samples working with the Flink 1.6?

Reply:
Yes, I have tested flink samples and it looks good.

Reply:
Great, in this case I approve the changes.